### DISCUSSION TITLE
DevCenter - Conteúdos em um único lugar

### DISCUSSION THREAD

**Discussion by oliveriojnn** [frontendbr/forum#678] [Discussão]
Pessoal, estou iniciando um projeto onde vai unir tudo quanto é 'conteúdo' que agrega valor para nossa área. Sei que já existe várias listas dessa no próprio github, tanto aqui no frontendbr, como no da galera. Mas a idéia é colocar em prática e acrescentar no portfólio. Caso dê certo, massa. Se não, fica pela experiência e portfólio! ;)

A idéia é unir tudo em um só lugar. Ser a central dos Developers! =)

Queria a opinião de vocês! O que vocês acharam do layout? Quais seriam as sugestões pra acrescentar?

Ps: Não sou bom em logo, e aí se caso alguém queira contribuir, entra em contato! 

![Home](https://s3.us-east-2.amazonaws.com/oliverio/home.png)
![Cursos](https://s3.us-east-2.amazonaws.com/oliverio/cursos.png)
![Podcasts](https://s3.us-east-2.amazonaws.com/oliverio/podcasts.png)
![Eventos](https://s3.us-east-2.amazonaws.com/oliverio/eventos.png)
![Meetups](https://s3.us-east-2.amazonaws.com/oliverio/meetups.png)
![Empregos](https://s3.us-east-2.amazonaws.com/oliverio/empregos.png)
![Blogs](https://s3.us-east-2.amazonaws.com/oliverio/blogs.png)

**Comment by woliveiras** [comment]
Fala, @oliveriojnn!

Eu não manjo muito de design, também, porém achei o layout legal. Talvez com o crescimento de conteúdo seja necessário dividir o que é curso/conteúdo free de pago, só para ficar mais fácil de quem quiser encontrar coisa free.

Uma dúvida: como existem várias listas no GitHub, não seria melhor criar um site para elas ou mesmo contribuir com os sites já existentes?

Ex.:

* para palestras: http://free-time.github.io/
* para podcasts: http://ogilvieira.com.br/awesome-podcasts-br/#/

**Comment by rubenmarcus** [comment]
O layout não está ruim mas remete muito a identidade visual do [PluralSight](https://www.pluralsight.com/)

**Comment by oliveriojnn** [comment]
Pô, não achei não. E nem tive ele como referência. 

Antes a cor tava roxa, mudei pra preto porque alguns lá no grupo Front-End falaram que o tom da cor tava muito forte.

**Comment by AmandaFonseca** [comment]
Eu gostei do Layout sobre a cor eu sempre prefiro temas escuros. Se eu tivesse  que mudar alguma coisa tiraria esse banner com o nome "Central dos developers" de cada divisão da página, ia deixar esse Banner só no começo.
Mas se quiser deixar sem problemas também parabéns pelo  trabalho

**Comment by raulfernando** [comment]
Seria uma boa colocar os títulos (Cursos, Podcasts, Eventos, Meetups, Empregos, Blogs) no lugar do Central dos developers. 

E na pagina blogs colocar o nome dos dev abaixo da foto ;)

**Comment by rubenmarcus** [comment]
dá pra encurtar o nome para Front-end Central ou Central Front-end se o público chave for só em Português.

**Comment by rubenmarcus** [comment]
1 - Eu registraria o dominio www.fecentral.com ou www.frontendcentral.com, está disponivel.
2 - faria um site com um layout parecido do seu, mas mais minimalista
3 - Tentaria agregar conteúdo ao site, fazer algo entre Tableless, Smashing Magazine, CSS Tricks, etc etc..

**Comment by WandersonAlves** [comment]
Acho que em vez de circulos para as listas (com exceção a parte de Blogs), poderia ser uma lista continua estilo cards do material sendo que pretos (e com umas customizações).

Tô com a visão disso na cabeça, mais tarde passo pra cá kkkk

**Comment by oliveriojnn** [comment]
@AmandaFonseca Também pensei em deixar esse banner só na home e tirar das páginas. Valeu! ;)

@rubenmarcus Curti muito os nomes não, kkkk. Pensei em algo relacionado a dev com brasil. Sim, também pensei na possibilidade de torná-lo um ponto de referência na área de conteúdo (blog próprio). Quem sabe, vamos ver como o projeto vai se sair.. ;)

@WandersonAlves Tem como desenhar em formato de wireframe, pelo menos? haha

Estou aberto a sugestões, pessoal! :)

**Comment by marquinhusgoncalves-zz** [comment]
Boa tarde !!! 
Gosto muito da ideia de sempre estar criando, inovando, fazendo coisas novas seja pra ajudar a si mesmo ou a comunidade, mas sigo na linha de pensamento do @woliveiras porque fazer mais um do que ajudar um outro que já existe, ou se caso não existir algo específico porque não agregar a algo que esta funcionando no caso desse Front End BR... Acho que seria mais válido usar sua energia para ajudar e/ou melhorar do que apenas criar mais um... 

Esse projeto aqui é exatamente o que você quer criar só falta atualizar, refazer o layout do site e darem continuidade e quem sabe até agregar aqui com o Front End BR. Da uma olhada.
http://cerebrobr.github.io/cerebro/
https://github.com/cerebrobr/cerebro

Algumas pessoas que já ajudaram no projeto:
@fdaciuk 
@zenorocha
@diogomoretti 
@vitorbritto 
Entre outros...

**Obs.:** O Projeto acima não é meu, mas quando descobri a uns anos achei a ideia excelente e sempre foi muito útil.
**Obs2.:** Não quero desmotiva-lo é apenas minha opinião, se for válida ok senão apenas desconsidere.

http://marquinhusgoncalves.com

**Comment by oliveriojnn** [comment]
Eu não conhecia o projeto dos caras. Realmente não tem pra que duplicar não.

Vou descontinuar esse aqui e tentar contribuir com esse.

Valeu! :)
